frontend/src/mocks/social.js

```
@@ -140,16 +140,21 @@ export const MOCK_AVAILABLE_NETWORKS = [
-// Re-export for backward compatibility
-export { TONE_EXAMPLES, TONE_OPTIONS };
-
 export const SHIELD_LEVELS = [
   { value: 90, label: '90% (Más laxo)' },
   { value: 95, label: '95%' },
   { value: 98, label: '98% (Más estricto)' },
   { value: 100, label: '100% (El más estricto)' },
 ];
 
+export const TONE_OPTIONS = [
+  'Flanders',
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TONE_OPTIONS'. Consider to delete it or rename it.

'TONE_OPTIONS' is defined here:


(lint/suspicious/noRedeclare)